TFBarcodeScanner/TFBarcodeScannerViewController.m

```diff
@@ -47,6 +47,22 @@ @interface TFBarcodeScannerViewController ()
 
 @implementation TFBarcodeScannerViewController
 
+@synthesize delegate;
+
+/*
++ (id<TFBarcodeScannerViewController>) delegate {
```

Comment:
There is no reason to synchronize access to the delegate. Let's get rid of these methods.